snp_motion_planning/CMakeLists.txt

```diff
@@ -28,15 +28,34 @@ find_package(tesseract_kinematics REQUIRED)
 find_package(LAPACK REQUIRED) # Requried for ikfast
 find_package(yaml-cpp REQUIRED)
 
+# Libraries
+add_library(
+  ${PROJECT_NAME}_tasks SHARED
+  src/plugins/tasks/constant_tcp_speed_time_parameterization_profile.cpp
+  src/plugins/tasks/constant_tcp_speed_time_parameterization_task.cpp
+  src/plugins/tasks/kinematic_limits_check_profile.cpp
+  src/plugins/tasks/kinematic_limits_check_task.cpp
+  src/plugins/tasks/tcp_speed_limiter_profile.cpp
+  src/plugins/tasks/tcp_speed_limiter_task.cpp)
+ament_target_dependencies(${PROJECT_NAME}_tasks ${ROS2_DEPS})
```

Comment:
```suggestion
```

Does this library actually need any ROS2 dependencies? I think it should just depend on `tesseract` libraries